client/src/App.jsx

```diff
@@ -1,27 +1,12 @@
-import logo from "./logo.svg";
-import "./App.css";
+import Create from "./components/Create";
```

Comment:
It would be best to have these existing in a `pages` directory like how was brought up in stand-up so when it comes to routing we would route to only existing pages, not components. Since these create pages will exist on a route.

Reply:
For reference, please take a look at Kevin's PR.